admin-notice/admin-notice.php

```diff
@@ -19,12 +19,14 @@
 add_action( 'admin_enqueue_scripts', [ $admin_notice_controller, 'enqueue_scripts' ] );
 add_action( 'wp_ajax_dismiss_vip_notice', [ $admin_notice_controller, 'dismiss_vip_notice' ] );
 
+// WP 5.6 RC1 (released Nov 17, 2020)
 $admin_notice_controller->add(
 	new Admin_Notice(
-		'WordPress 5.5.2 will be released on Friday, October 30th',
+		'WordPress 5.6 will be released on December 8th, 2020. Please ensure your sites have been tested against the current Release Candidate',
```

Comment:
Can we add a link to the Lobby post or something?

Reply:
Yep the previous wording was just a placeholder until the Lobby post went live.